core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -164,11 +171,30 @@ export class CreateServiceComponent implements OnInit {
 
   // function for changing platform type
   changePlatformType(platformType){
-    if(!this.disablePlatform){
+    if(!this.disablePlatform && platformType !== "gcloud"){
```

Comment:
`!this.disablePlatform` should cover this. Why do we need to make an explicit check on `gcloud`?

Reply:
@oscarbrooksdev9

Reply:
If we remove this the gcloud option becomes clickable like you mentioned you wanted it fixed a while ago. Fixing it this way allows so simply remove the gcloud check in the future to enable it. If we did it another way we would have to introduce major changes, and then someone would have to undo those changes later when gcloud would become available.